### DISCUSSION TITLE
📣 POLL: What programming language do you use for your SP-API application?

### DISCUSSION THREAD

**Discussion by rodrifed** [amzn/selling-partner-api-samples#37] [Polls]
The SP-API team is looking for your feedback in order to identify the resources that would help you the most in your API integration journey.
Please answer the poll below, raise your voice, and help us shape the future of SP-API content!

**Comment by johnkw** [comment]
Thanks, but you can write an example workflow in any language. Users will be able to translate it into their language of choice.
